build/depends.py

```diff
@@ -430,8 +436,11 @@ def configure(self, build, conf):
 
         # Mixxx requires C++11 support. Windows enables C++11 features by
         # default but Clang/GCC require a flag.
+        #
+        # This needs to be changed to -std=c++14 to support std::make_unique
+        # as requested by daschuer (https://github.com/mixxxdj/mixxx/pull/2078#discussion_r272895681)
         if not build.platform_is_windows:
-            build.env.Append(CXXFLAGS='-std=c++11')
+            build.env.Append(CXXFLAGS='-std=c++14')
```

Comment:
Ups, sorry I should have say that we have our own implementation: 
src/util/memory.h that works with c++11

However maybe it is time to enable c++14, but that should be done in a separate PR.

Reply:
That's fine. Thanks for mentioning it! I will switch to src/util/memory.h if possible